src/layouts/partials/help_feedback.html

```diff
@@ -1,6 +1,6 @@
 {{ with .File }}
 <div class="feedback well">
   <h5><i class="fa fa-help-outline"></i> Need help, got feedback?</h5>
-  <p>We listen in your Slack support channel. You can also reach us at <a href="mailto:support@giantswarm.io">support@giantswarm.io</a>. And of course, we welcome your <a title="open this page in GitHub" href="https://github.com/giantswarm/docs/blob/master/src/content/{{ .Path }}" target="_blank" rel="noopener noreferrer">pull requests</a>!</p>
+  <p>We listen to your Slack support channel. You can also reach us at <a href="mailto:support@giantswarm.io">support@giantswarm.io</a>. To continue the conversation, join us at our <a href="https://swarm.ly/37lDbAj>monthly Office Hour</a>. And of course, we welcome your <a title="open this page in GitHub" href="https://github.com/giantswarm/docs/blob/master/src/content/{{ .Path }}" target="_blank" rel="noopener noreferrer">pull requests</a>!</p>
```

Comment:
You were just shy of a single `"` quote :) I've also added opening the link in a new tab instead of the current one.

```suggestion
  <p>We listen to your Slack support channel. You can also reach us at <a href="mailto:support@giantswarm.io">support@giantswarm.io</a>. To continue the conversation, join us at our <a href="https://swarm.ly/37lDbAj" target="_blank" rel="noopener">monthly Office Hour</a>. And of course, we welcome your <a title="open this page in GitHub" href="https://github.com/giantswarm/docs/blob/master/src/content/{{ .Path }}" target="_blank" rel="noopener noreferrer">pull requests</a>!</p>
```